src/app/state/playback_state.rs

```diff
@@ -314,36 +311,39 @@ impl PlaybackState {
         self.is_playing = false;
     }
 
+    fn play_index(&mut self, index: Option<usize>) -> Option<String> {
```

Comment:
nitpick: callers of play_index should have to handle the Option and unwrap it somehow, since there's no point in calling play_index(None)

Reply:
good idea otherwise to have this function!